.circleci/config.yml

```diff
@@ -22,7 +21,49 @@ jobs:
           paths:
             - ~/.m2
           key: v1-dependencies-{{ checksum "build.gradle" }}
+  run-api-diff:
+    steps:
+      - checkout
+      - run: chmod +x gradlew
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "build.gradle" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+      # run apiDiff task
+      - run: ./gradlew clean apiDiff
+      - store_artifacts:
+         path: build/reports/apiDiff/apiDiff.txt
```

Comment:
I thought the output was going to be HTML. Can we upload that one as well? as it's easier to browse through that report vs plain text

Reply:
Uploading both html and txt reports now.